hack/move-upstream.sh

```diff
@@ -30,7 +35,7 @@ if [[ -z "${NO_REBASE-}" ]]; then
   lastrev="$(go run ${OS_ROOT}/tools/godepversion/godepversion.go ${OS_ROOT}/Godeps/Godeps.json ${repo}/${package})"
 fi
 
-branch="$(git rev-parse --abbrev-ref HEAD)"
+branch="${TARGET_BRANCH:-$(git rev-parse --abbrev-ref HEAD)}"
```

Comment:
Because you're working from master in Origin?  I normally invoke this from a branch, so the branch in kube is created to equal it